instructor/common.py

```diff
@@ -667,6 +667,61 @@ def western_blot_band_size(request):
         context_instance=RequestContext(request))
 
 
+def western_blot_band_intensity(request):
+    pk = request.session['assignment_id']
+    assignment = models.Assignment.objects.get(id=pk)
+    wb, created = models.WesternBlot.objects.get_or_create(assignment=assignment)
+    BandsFormset = modelformset_factory(
+        models.WesternBlotBands,
+        extra=0,
+        exclude=['strain_protocol', 'antibody', 'weight', 'lysate_type']
+    )
+    # Create Bands
+    antibodies = models.WesternBlotAntibody.objects.filter(western_blot=wb)
+    strain_treatments = models.StrainTreatment.objects.filter(assignment=assignment)
+    weights_by_type = ['wc_weight', 'nuc_weight', 'cyto_weight']
+    for antibody in antibodies:
+        for strain_treatment in strain_treatments:
+            for index, field in enumerate(weights_by_type):
+                weight_str = getattr(antibody, field)
+                weights = [int(s) for s in weight_str.split(',') if s.isdigit()]
```

Comment:
If there's a space in `s` then `isdigit` will reject it. Can you add a `.strip()` to remove whitespace?